webapp/app/templates/basis/base.html

```diff
@@ -78,10 +79,14 @@
             {% endblock %}
             {% block main_content %}
                 <div class="main-content">
+                    {% if render_info and render_info.prev_url %}
                     <div class="header-navigation">
-                        {% block header_navigation %}
-                        {% endblock %}
+                        <div class="mt-3">
```

Comment:
I moved that from the marcos into the base.html. Its the only use and I don't want to render the header navigation if the content is empty.